doc/dev/custom_devices.rst

```diff
@@ -194,6 +196,33 @@ The ``get_c_interface`` method should be a static method that takes no parameter
     def f():
         return measure(0)
 
+or with the new device API:
+
+.. code-block:: python
+
+    class CustomDevice(qml.devices.Device):
```

Comment:
Are there any special considerations we should point out about what used to be `device.operations` and is now handled by `device.preprocess`, or will Catalyst generate the preprocess function for devices based on the TOML?
If not, how do we intend Catalyst to take influence on the preprocessing stage?

Reply:
@rmoyard can we utilize the TOML validation/processing to do both with one stone?

The main thinking with the TOML files was thinking forward to the new device API, that doesn't include the ability to specify this in python

Reply:
Yes I will create transforms (preprocessing) that use the operations/observables from TOML files.